### PR TITLE
⚠️ Experiments ⚛️ DONT MERGE

### PR DESCRIPTION
**Pull request by andresfulla**
@sandrodw3

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/ConnectedComponent.es.js [hunk 1]
```diff
@@ -88,5 +88,54 @@ const getConnectedComponent = (Component, properties) => {
...
+					this.handleChange.bind(this)
+				);
+			}
+
+			componentWillUnmount() {
+				this._subscriber.removeListener();
+			}
+
+			handleChange() {
+				// TODO: try to avoid `forUpdate()`
```
Comment (andresfulla) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/store/ConnectedComponent.es.js]: Any ideas here?

**Comment by p2kmgcl** [comment]
Use proper ascii old style emoticons instead of emojis.
Please resend.

**Comment by p2kmgcl** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by andresfulla** [comment]
Closing, I had to integrate with my _backenders_. Pull goes on here https://github.com/darquesdev/liferay-portal/pull/61

**Pull request closed without merging**